Common/include/config_structure.hpp

```diff
@@ -9354,6 +9355,12 @@ class CConfig {
    * \return <TRUE> if option was set in the config file
    */
   bool OptionIsSet(string option);
+  
+  /*!
+   * \brief Get the name of the current case
+   * \return the case name
+   */
+  string GetCaseName();
```

Comment:
yeah probably. Didn't think about it since no other routine in CConfig has it so far. I can change that. We have remember to do it when restructuring.